docs/getting-started/fundamentals/block-json.md

```diff
@@ -60,7 +63,7 @@ _Example: Attributes as defined in block.json_
 	}
 },
 ```
-By default `attributes` are serialized and stored in the block's delimiter but this [can be configured](https://developer.wordpress.org/news/2023/09/understanding-block-attributes/).
+By default `attributes` are serialized and stored in the block's delimiter, but this [can be configured](https://developer.wordpress.org/news/2023/09/understanding-block-attributes/).
```

Comment:
If we aren't talking about the property specifically, I am not sure "attributes" needs to be inline-code.
```suggestion
By default, attributes are serialized and stored in the block's delimiter, but this [can be configured](https://developer.wordpress.org/news/2023/09/understanding-block-attributes/).
```